telemetry/specs/INT.mdk

```diff
@@ -1407,6 +1585,21 @@ INT Metadata Header and Metadata Stack, followed by TCP header:
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
```

Comment:
IMO it would be cleaner to add a section header. Change the title of the previous example, replacing "INT" with "INT-MD", then add a title here with "INT-MX", referring to the previous section for the IP and TCP headers. Start with the INT shim header with type = 3 and length = 3 here.

Reply:
Done